GOVERNANCE.md

```diff
@@ -9,9 +9,21 @@ The [nodejs/node](https://github.com/nodejs/node) GitHub repository is
 maintained by Collaborators who are added by the CTC on an ongoing basis.
 
 Individuals identified by the CTC as making significant and valuable
-contributions are made Collaborators and given commit access to the project. If
-you make a significant contribution and are not considered for commit access,
-log an issue or contact a CTC member directly.
+contributions across any Node.js repository may be made Collaborators and given
+commit access to the project. Activities taken into consideration include (but
+are not limited to) the quality of:
+
+* code commits and pull requests
+* documentation commits and pull requests
+* comments on issues and pull requests
+* contributions to the Node.js website
+* assistance provided to end users and novice contributors
+* participation in Working Groups
+* other participation in the wider Node.js community
```

Comment:
AIUI "commit access to the project" does not mean "commit access to nodejs/node", it means commit access to whichever team you get added to. So if you contribute to core you get made a core collaborator and added to `@nodejs/core` (assuming we change `@nodejs/collaborators` to that). If you contribute to the website, then you become a website collaborator and get website access.

To clarify, maybe something like:

```diff
-contributions across any Node.js repository may be made Collaborators and given
-commit access to the project. Activities taken into consideration include (but
+contributions across any Node.js repository may be made Collaborators and given
+commit access to that repository. Activities taken into consideration include (but
```